app/models/peep.rb

```diff
@@ -0,0 +1,11 @@
+require 'dm-validations'
+
+class Peep
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :message, Text
+  property :created_at, DateTime
+
+  has 1, :user, :through => Resource
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)